src/util.ts

```diff
@@ -393,3 +400,43 @@ const addGraphTotals = (
   )
   return { totalTxs, totalUsd }
 }
+
+export const movingAveDataSort = (
+  data: Data[]
+): Array<{ date: string; allTxs: number }> => {
+  const aveArray: Array<{ date: string; allTxs: number }> = []
+  if (Array.isArray(data) === false) return aveArray
```

Comment:
This is actually one of the only places where an implicit check is fine since IT IS a boolean:
`if (!Array.isArray(data)) return aveArray`

Reply:
Made changes according to comment.